### PR TITLE
Fixing remark-lint issues. Adding remark-linter check.

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Summary
Based on the check settings for remark-lint in the PR #316 , a revision was made .md files that contained warnings.
I combined 2 PRS in a separate branch and checked it in my fork: https://github.com/dvkruchinin/datumaro/pull/8

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md)
- [ ] I have updated the [documentation](
  https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have [linked related issues](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -53,51 +53,57 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Support for Validator configurable threshold (<https://github.com/openvinotoolkit/datumaro/pull/250>)
 
 ### Changed
-- LabelMe format saves dataset items with their relative paths by subsets without changing names (<https://github.com/openvinotoolkit/datumaro/pull/200>)
-- Allowed arbitrary subset count and names in classification and detection splitters (<https://github.com/openvinotoolkit/datumaro/pull/207>)
+- LabelMe format saves dataset items with their relative paths by subsets
+without changing names (<https://github.com/openvinotoolkit/datumaro/pull/200>)
```
Comment (IRDonch) [inline CHANGELOG.md:57]: Please align continuation lines with the first line, e.g.:

```suggestion
- LabelMe format saves dataset items with their relative paths by subsets
  without changing names (<https://github.com/openvinotoolkit/datumaro/pull/200>)
```

File: docs/design.md [hunk 2]
```diff
@@ -65,15 +66,16 @@ Datumaro is:
...
+  with lesser training time
 - dataset expansion and filtration of additions
   Use case: add only important data
 - guidance for key frame selection for tracking ([paper](https://arxiv.org/abs/1903.11779))
   Use case: more effective annotation, better predictions
 
 ## RC 1 vision
 
-*CVAT integration*
+_CVAT integration_
```
Comment (IRDonch) [inline docs/design.md:78]: IMO, this should be a 3rd-level heading instead.

**Comment by IRDonch** [comment]
I think it would be good to move the remark workflow addition to this PR, so that:

1. All the fixes are validated.
2. Nobody can introduce new issues before the workflow is added.

File: CHANGELOG.md (hunk 1 above)
Comment (dvkruchinin) [reply CHANGELOG.md]: Done

File: docs/design.md (hunk 2 above)
Comment (dvkruchinin) [reply docs/design.md]: Done

**Comment by dvkruchinin** [comment]
> I think it would be good to move the remark workflow addition to this PR, so that:
> 
> 1. All the fixes are validated.
> 2. Nobody can introduce new issues before the workflow is added.

Combined.

File: .github/workflows/remark.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+            node-version: 12
+
+      - name: Run checks
+        run: |
+          npm ci
+          mkdir -p remark_report
```
Comment (IRDonch) [inline .github/workflows/remark.yml:15]: This is now unneeded.

Comment (dvkruchinin) [reply .github/workflows/remark.yml]: Sure. Removed.

**Comment by dvkruchinin** [comment]
> > > I think it would be good to move the remark workflow addition to this PR, so that:
> > > 
> > > 1. All the fixes are validated.
> > > 2. Nobody can introduce new issues before the workflow is added.
> > 
> > 
> > Combined.
> 
> I meant to only add the _remark_ workflow, not Pylint or Bandit. This PR only fixes remark issues, after all.

I think can leave it like this.

**Comment by IRDonch** [comment]
> I think can leave it like this.

And I think you can't. Adding the remark wokrflow is safe, because this PR fixes all existing remark issues in the codebase. The same is not true for Pylint and Remark. If those pipelines are added before the existing issues are fixed, pull requests may start to fail due to unrelated issues.

**Comment by dvkruchinin** [comment]
> > I think can leave it like this.
> 
> And I think you can't. Adding the remark wokrflow is safe, because this PR fixes all existing remark issues in the codebase. The same is not true for Pylint and Remark. If those pipelines are added before the existing issues are fixed, pull requests may start to fail due to unrelated issues.

Done. PR #316 re-opened.

**Review by IRDonch — APPROVED** [review]

**Pull request merged**